lib/cisco_node_utils/platform.rb

```diff
@@ -164,7 +164,7 @@ def self.virtual_services
       # convert to expected format
       virts_hsh = {}
       virts.each do |serv|
-        # rubocop:disable Style/AlignHash, Style/ExtraSpacing
+        # rubocop:disable Style/AlignHash
```

Comment:
I was trying to make this a pre-emptive fix for 0.35 but several of these changes are not compatible with 34.2 so I'm reverting the rubocop changes I made for this diff. I'll resubmit with just the minitest fixes.